src/mach/imports.rs

```diff
@@ -195,9 +216,9 @@ impl<'a> BindInterpreter<'a> {
 	            // throwBadBindingAddress(address, segmentEndAddress, segmentIndex, start, end, p);
 	            // (this->*handler)(context, address, type, symbolName, symboFlags, addend, libraryOrdinal, "", &last);
 	            // address += sizeof(intptr_t);
+                    imports.push(Import::new(&bind_info, libs, segments, start_of_sequence));
                     let seg_offset = bind_info.seg_offset.wrapping_add(ctx.size() as u64);
                     bind_info.seg_offset = seg_offset;
-                    imports.push(Import::new(&bind_info, libs, segments));
```

Comment:
Yea wow this was horrible... thanks again for fixing this!

I've been testing on `libssl.dylib` (64 bit version sucked out), and, for example, old version gives wrong address for `_time`, so when implementing this line https://github.com/das-labor/panopticon/pull/321/commits/597b1d581f3272338231950887c1b03041c9ff1f#diff-4c438f60247f494e2aee67fcce9e11e0R112

since i was receiving lies from goblin and consequently it says that nothing calls `_time`; but newer fixed version I believe outputs correct values (can't check with a debugger though, since not on an osx system)